CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 0.0.22 (Unreleased)
```

Comment:
@mavarius and I just fixed some changelog / versioning wonkiness, so you'll probably need to rebase this against master and fix the conflict before merging.